src/Components/Search/SearchBar.tsx

```diff
@@ -441,14 +440,21 @@ export const SearchBarQueryRenderer: React.FC = () => {
       render={({ props }) => {
         if (props) {
           return <SearchBarRefetchContainer viewer={props.viewer} />
+          // SSR render pass. Since we don't have access to `<Boot>` context
+          // from within the NavBar (it's not a part of any app) we need to lean
+          // on styled-system for showing / hiding depending upon breakpoint.
         } else {
           return (
-            <Input
-              name="term"
-              style={{ width: "100%" }}
-              placeholder={PLACEHOLDER_XS}
-            />
+            <>
+              <Box display={["block", "none"]}>
+                <SearchInputContainer placeholder={PLACEHOLDER_XS} />
+              </Box>
+              <Box display={["none", "block"]}>
+                <SearchInputContainer placeholder={PLACEHOLDER} />
+              </Box>
+            </>
```

Comment:
@mzikherman / @pepopowitz - this fixes the weird SearchBar missing icon / text issue during the SSR render pass, when the searchbar first mounts: 

![glass](https://user-images.githubusercontent.com/236943/58145377-b0d66780-7c06-11e9-9c7d-a3a5ac44577e.gif)